components/ILIAS/Authentication/classes/class.ilAuthPageEditorSettings.php

```diff
@@ -18,7 +18,7 @@
 
 declare(strict_types=1);
 
-class ilAuthLogoutPageEditorSettings
+class ilAuthPageEditorSettings
 {
     private static ?self $instance = null;
```

Comment:
If we support multiple contexts, we need an array of `$instances`.

```suggestion
/** @var array<value-of<AuthPageEditorContext>, self> */
private static array $instances = [];
```